packages/errors/src/codes.ts

```diff
@@ -5,30 +5,46 @@
+ *     - `_MISSING`: Some required input is missing. E.g. `SUBTLE_CRYPTO_VERIFY_FUNCTION_MISSING`.
+ *     - `_MUST_BE_`: Some condition must be try. E.g. `NONCE_TRANSACTION_FIRST_INSTRUCTION_MUST_BE_ADVANCE_NONCE`.
  */
-export const SOLANA_ERROR__TRANSACTION_MISSING_SIGNATURES = 1 as const;
-export const SOLANA_ERROR__TRANSACTION_SIGNATURE_NOT_COMPUTABLE = 2 as const;
+export const SOLANA_ERROR__TRANSACTION_SIGNATURES_MISSING = 1 as const;
+export const SOLANA_ERROR__TRANSACTION_CANNOT_COMPUTE_SIGNATURE = 2 as const;
 export const SOLANA_ERROR__RPC_INTEGER_OVERFLOW = 3 as const;
 export const SOLANA_ERROR__INVALID_KEYPAIR_BYTES = 4 as const;
 export const SOLANA_ERROR__BLOCK_HEIGHT_EXCEEDED = 5 as const;
-export const SOLANA_ERROR__NONCE_INVALID = 6 as const;
+export const SOLANA_ERROR__INVALID_NONCE = 6 as const;
 export const SOLANA_ERROR__NONCE_ACCOUNT_NOT_FOUND = 7 as const;
 export const SOLANA_ERROR__ACCOUNT_NOT_FOUND = 8 as const;
 export const SOLANA_ERROR__MULTIPLE_ACCOUNTS_NOT_FOUND = 9 as const;
 export const SOLANA_ERROR__FAILED_TO_DECODE_ACCOUNT = 10 as const;
 export const SOLANA_ERROR__EXPECTED_DECODED_ACCOUNT = 11 as const;
-export const SOLANA_ERROR__NOT_ALL_ACCOUNTS_DECODED = 12 as const;
-export const SOLANA_ERROR__INCORRECT_BASE58_ADDRESS_LENGTH = 13 as const;
-export const SOLANA_ERROR__INCORRECT_BASE58_ADDRESS_BYTE_LENGTH = 14 as const;
-export const SOLANA_ERROR__NOT_A_BASE58_ENCODED_ADDRESS = 15 as const;
-export const SOLANA_ERROR__NOT_AN_ED25519_PUBLIC_KEY = 16 as const;
-export const SOLANA_ERROR__MALFORMED_PROGRAM_DERIVED_ADDRESS = 17 as const;
-export const SOLANA_ERROR__PROGRAM_DERIVED_ADDRESS_BUMP_SEED_OUT_OF_RANGE = 18 as const;
+export const SOLANA_ERROR__EXPECTED_ALL_ACCOUNTS_TO_BE_DECODED = 12 as const;
+export const SOLANA_ERROR__INVALID_BASE58_ADDRESS_LENGTH = 13 as const;
+export const SOLANA_ERROR__INVALID_BASE58_ADDRESS_BYTE_LENGTH = 14 as const;
+export const SOLANA_ERROR__MALFORMED_BASE58_ENCODED_ADDRESS = 15 as const;
+export const SOLANA_ERROR__MALFORMED_ED25519_PUBLIC_KEY = 16 as const;
+export const SOLANA_ERROR__MALFORMED_PDA = 17 as const;
+export const SOLANA_ERROR__PDA_BUMP_SEED_OUT_OF_RANGE = 18 as const;
 export const SOLANA_ERROR__MAX_NUMBER_OF_PDA_SEEDS_EXCEEDED = 19 as const;
 export const SOLANA_ERROR__MAX_PDA_SEED_LENGTH_EXCEEDED = 20 as const;
 export const SOLANA_ERROR__INVALID_SEEDS_POINT_ON_CURVE = 21 as const;
-export const SOLANA_ERROR__COULD_NOT_FIND_VIABLE_PDA_BUMP_SEED = 22 as const;
-export const SOLANA_ERROR__PROGRAM_ADDRESS_ENDS_WITH_PDA_MARKER = 23 as const;
+export const SOLANA_ERROR__FAILED_TO_FIND_VIABLE_PDA_BUMP_SEED = 22 as const;
+export const SOLANA_ERROR__PDA_CANNOT_END_WITH_PDA_MARKER = 23 as const;
 export const SOLANA_ERROR__SUBTLE_CRYPTO_MISSING = 24 as const;
 export const SOLANA_ERROR__SUBTLE_CRYPTO_DIGEST_MISSING = 25 as const;
 export const SOLANA_ERROR__SUBTLE_CRYPTO_ED25519_ALGORITHM_MISSING = 26 as const;
```

Comment:
Still sort of prefer `UNIMPLEMENTED` here rather than ‘missing.’ Something about ‘missing’ doesn't feel right.